src/queue/multilevel.rs

```diff
@@ -315,6 +362,7 @@ impl LevelManager {
         };
         extras.current_level = current_level;
         extras.schedule_time = Some(now());
+        self.tasks_per_level[current_level as usize].inc();
```

Comment:
This is a global atomic value that is mutated very frequently. I'm thinking whether this can be avoided by making this a [Collector](https://docs.rs/prometheus/latest/prometheus/core/trait.Collector.html#). It can be a wrapper of a normal `Gauge` but we can update it lazily.

So, only when Prometheus pulls the data or when we want to use this to adjust some internal parameters, we call `Injector::len` to get the length to avoid extra atomic operations.

Reply:
Yeah, use a Collector is a good idea here. Changing to collecor need a global lazy static variable to store a weak ref of all running injectors, this need to change some of the structs. But I found we already had `tikv_unified_read_pool_running_tasks`, so no need to add another similar metrics. I'm going to remove this metric anyway.